components/script/dom/htmliframeelement.rs

```diff
@@ -354,10 +354,12 @@ impl MozBrowserEventDetailBuilder for HTMLIFrameElement {
                 rval.set(NullValue());
             }
             MozBrowserEvent::Error(error_type, description, report) => {
+                let version = format!("Servo {}{}", env!("CARGO_PKG_VERSION"), env!("GIT_INFO"));
```

Comment:
This could get out of sync with the version string in `components/servo/main.rs` - can we define it in one place and reference it from both? (Maybe `util`, not sure which crate is right here.)